### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by msupernaut**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

Testing is an important discipline in DevSecOps. I've curated a collection of open source tools and frameworks that helps testing teams to ship code faster. These are frameworks like Selenium, Appium , Playwright, Cypress etc that are widely used by OSS teams and enterprises alike - as well as tools that help create/write test cases. 

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [x] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/OpenSource-Testing/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ - microsoft/playwright
+ - cypress-io/cypress
+ - casperjs/casperjs
+ - webdriverio/webdriverio
+ - joelgriffith/navalia
+ - webcatalog/chromeless
+ - laurentj/slimerjs
+ - sdesalas/trifleJS
+ - karma-runner/karma
+ - eWert-Online/osnap
```
Comment (kenyonj) [inline collections/OpenSource-Testing/index.md:14]:
```suggestion
 - eWert-Online/OSnap
```

File: collections/OpenSource-Testing/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ - katalon-studio
+ - qawolf/qawolf
+ - basset/basset
+ - newsuk/ayespy
```
Comment (kenyonj) [inline collections/OpenSource-Testing/index.md:18]:
```suggestion
 - newsuk/AyeSpy
```

File: collections/OpenSource-Testing/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ - angular/protractor
+ - Hapag-Lloyd/jest-puppeteer-react
+ - americanexpress/jest-image-snapshot
+ - selenide/selenide
+ - trytouca/trytouca
+ - HuddleEng/PhantomCSS
+display_name: OpenSource Testing Frameworks & Tools
+created_by: msupernaut
+---
+Open source projects to help testers/QA teams with testing tools and frameworks to automate software delivery with high quality
```
Comment (kenyonj) [inline collections/OpenSource-Testing/index.md:66]: This looks like an incomplete sentence. What do you think about this?
```suggestion
Open source testing tools and frameworks to help testers/QA teams with automating and delivering high quality software.
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
Thanks for the PR. Couple of comments. Unfortunately our current implementation is case sensitive so we'll need to update a couple of the repo references.

**Comment by msupernaut** [comment]
Thank you ! Made the updates !

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Pull request merged**